### PR TITLE
Forkserver builder fix

### PR DESCRIPTION
**Pull request by tokatoka**
I noticed there's a buggy pattern

### CODE REVIEW

**Comment by domenukk** [comment]
maybe for the command line, but not for the API

File: libafl/src/executors/forkserver.rs [hunk 1]
```diff
@@ -560,11 +560,23 @@ impl<'a, SP> ForkserverExecutorBuilder<'a, SP> {
         };
 
         for item in &self.arguments {
+            // need special handling for @@
```
Comment (domenukk) [inline libafl/src/executors/forkserver.rs:563]: I would propose to not add this to the builder but have something like this:
https://github.com/AFLplusplus/LibAFL/blob/9d38fff662f4647117dff841fb9e8b30bc7ae5fe/libafl/src/executors/command.rs#L265

Comment (domenukk) [reply libafl/src/executors/forkserver.rs:563]: We could even potentially share some code here 🤔

Comment (tokatoka) [reply libafl/src/executors/forkserver.rs]: so you mean separating this code from here and put it into another function in `ForkserverExecutorBuilder`?

Comment (tokatoka) [reply libafl/src/executors/forkserver.rs]: ah, ok then users can choose if they want the AFL-style parsing or not

**Comment by andreafioraldi** [comment]
https://github.com/AFLplusplus/LibAFL/blob/main/fuzzers/backtrace_baby_fuzzers/forkserver_executor/src/main.rs#L89

This fuzzer is missing in main

**Pull request merged**